libfreerdp/crypto/test/Test_x509_cert_info.c

```diff
@@ -19,7 +19,36 @@ char*   crypto_cert_subject_common_name_wo_length(X509* xcert)
 	return crypto_cert_subject_common_name(xcert, & length);
 }
 
-const char* certificate_path = "Test_x509_cert_info.pem";
+const char* certificate_path()
```

Comment:
Possible, easier would be to add `add_definition(-DFOO="${CMAKE_CURRENT_SOURCE_DIR}")` in `CMakeLists.txt` and do a `const char* certificate_path=FOO"/Test_x509_cert_info.pem";`